File: MODUL_5/C/PRAK502_2310817310014_MUHAMMADRIZKISAPUTRA.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2)
{
    int hasil = nilai1 - nilai2;
    return (hasil < 0) ? -hasil : hasil;
}

int mutlak(int angka)
{
    return (angka < 0) ? -angka : angka;
}

int main()
{
    int a, b, c, d, Hasil;
    scanf("%d", &a);
    scanf("%d", &c);
    scanf("%d", &b);
    scanf("%d", &d);
    Hasil = hitung(a, b) + hitung(c, d);
    printf("%d", Hasil);
    return 0;
}

File: MODUL_5/Py/PRAK502_2310817310014_MUHAMMADRIZKISAPUTRA.py
def hitung(nilai1, nilai2):
    return abs(nilai1 - nilai2)

def mutlak(angka):
    return abs(angka)

input_baris = input().split()
a, c, b, d = map(int, input_baris)

hasil = hitung(a, b) + hitung(c, d)
print(hasil)
